test/gluttony_test.exs

```diff
@@ -37,8 +38,21 @@ defmodule GluttonyTest do
     test "TechCrunch", %{xml4: xml} do
       assert {:ok, %{feed: feed, entries: entries}} = Gluttony.parse_string(xml)
       assert feed.title == "TechCrunch"
+      # dc:creator element test, agnostic to item list order, either first and last items' authors can match
+      assert List.first(entries).author in ["Amanda Siberling", "Sarah Perez"]
       assert Enum.count(entries) == 20
     end
+
+    test "NJ Monitor (Ghost)", %{xml5: xml} do
```

Comment:
Same here, the tests in this file are more to guarantee that we have the correct structure and that the number of entries after parsing is correct.

Let's create a test for this handler as well. However, I'm not sure we should be asserting that the content is longer than the description because we can't enforce this. I think is better if we just test if the structure is correct. 

Because the raw map will now return both `description` and `content`, we should test that when we use `raw: false` option, the final `Gluttony.Entry` struct will prefer `content:encoded` over description when provided.